pilot/cmd/pilot-agent/main.go

```diff
@@ -97,11 +97,11 @@ var (
 	provCert = env.RegisterStringVar("PROV_CERT", "",
 		"Set to a directory containing provisioned certs, for VMs").Get()
 
-	// set to "/etc/ssl/certs/ca-certificates.crt" on debian/ubuntu for ACME/public signed XDS servers.
+	// set to "SYSTEM" for ACME/public signed XDS servers.
```

Comment:
Is this change for 1.8 or 1.9?

If it is 1.9, any consideration for backwards compatible?  If operator sets this to `/etc/ssl/certs/ca-certificates.crt` for istio 1.8.

Reply:
Either one? it's backwards compatible, they can still set explicit file path

Reply:
Probably 1.9 at this point, a bit late for 1.i

Reply:
Thanks John, SG!